src/test/java/example/tri/stringreverse/StringReverseTest.java

```diff
@@ -0,0 +1,13 @@
+package example.tri.stringreverse;
+
+import tri.stringreverse.StringReverse;
+
+/**
+ * Created by TriTruong on 7/12/2015.
+ */
+public class StringReverseTest {
+    public void stringreversed(){
```

Comment:
To make this method stringreversed() method a test method, you need to add this annotation @Test (and import org.unit.Test, IntelliJ will help you to import the class). Once you make this method become a test method, learn to run the test (by right click on the whole class and run test).